app/pages/homepage/ui.test.js

```diff
@@ -1,51 +1,60 @@
 import nock from 'nock';
-import { Selector, ClientFunction } from 'testcafe';
+import { Selector, ClientFunction, RequestLogger } from 'testcafe';
 import content from './manifest.json';
 import { extractInnerText } from '../../test-utils/helper';
 
-const setCookies = ClientFunction(() => {
+const getLocation = ClientFunction(() => document.location.href);
+
+const setfakeTokenCookie = ClientFunction(() => {
   const cookieValue = JSON.stringify({
-    id: '88421113', name: {}, _raw: '{"sub":"88421113"}', _json: { sub: '88421113' },
+    id: '88421113', name: 'Cool Dude', _raw: '{"sub":"88421113"}', _json: { sub: '88421113' },
   });
-
   document.cookie = `fakeToken=${cookieValue}`;
 });
 
-const pageSetup = async (t, withAuth = false) => {
-  if (withAuth) {
-    await setCookies();
-  }
+const setLoggedInCookies = ClientFunction(() => {
+  document.cookie = 'cookie1=cookie1;';
+});
+
+const getCookies = ClientFunction(() => document.cookie);
+
+const pageSetup = async ({ t, withAuth = false, withLoggedInCookies = false }) => {
```

Comment:
I don't think we need both here. They are both serving the same purpose which is setting a cookie on the document to replicate being logged in